swap/src/bin/asb.rs

```diff
@@ -63,8 +63,9 @@ async fn main() -> Result<()> {
     };
 
     info!(
-        "Database and Seed will be stored in directory: {}",
-        config.data.dir.display()
+        db_folder = %config.data.dir.display(),
+        "Database and Seed will be stored in",
+
     );
```

Comment:
I think for the CLI, we removed that line entirely because it is redundant with what is logged while reading the config.